state_machine/applications/system-check/tests/coil_test.py

```diff
@@ -0,0 +1,130 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Torque Driver Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from lib import pycubed as cubesat
+from ulab import numpy
+
+# voltage level constants; set between -1 and 1
+v1 = 0.25
+v2 = 0.5
+v3 = 0.75
+
+# find projected voltage: vlevel * max voltage
+projected_voltage1 = v1 * 5.06
+projected_voltage2 = v2 * 5.06
+projected_voltage3 = v3 * 5.06
+
+# alias norm
+norm = numpy.linalg.norm
+
+
+def test_voltage_levels(coil_index):
+    """
+    All user interaction happens in this function
+    Set wait times, prompt user, collect magnetometer data for
+    different voltage levels
+    """
+
+    # wait times
+    wait_time = 3
+    driver_time = 3
+
+    # set up the test
+    print(f'Testing Coil Driver {coil_index} for the following voltage ' +
```

Comment:
Use `f"""string contents"""` 
The triple quotes allow you to do multi-line strings (like in the doc strings)

Reply:
thanks! fixed